src/components/radio/radio_group.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
@jelbourn sorry for the noise but I'm really curious how you could add `<ng-content>` with a `@Directive()` any example pointer? This then would be the same here https://github.com/angular/material2/blob/4009fb965dc5ce157c5bccaa6a879e29ef218712/src/components/sidenav/sidenav.ts#L49 ?